projects/gnutls/project.yaml

```diff
@@ -1,6 +1,6 @@
 homepage: "https://www.gnutls.org"
 language: c++
-primary_contact: "n.mavrogiannopoulos@gmail.com"
+primary_contact: "daiki.ueno@gmail.com"
 auto_ccs:
   - "daiki.ueno@gmail.com"
```

Comment:
Probably want to change this auto_cc to n.mavrogiannopoulos@gmail.com ?

Reply:
I do not think that's necessary.

Reply:
Ok can you remove the duplicate email from auto_ccs field.